### PR TITLE
bugfix: VimNavigation on Input Fields

### PR DESCRIPTION
**Pull request by sshanzel**
As stated in the [issue](https://github.com/dailydotdev/daily/issues/139), input fields malfunction (other than the `feed` `searchbox`). Due to the reason that the keypress Navigation was tagged on `window`.

For the resolution, i've thought of having a trigger point where components can disable the Navigation truly whenever necessary (which will then be enabled based on the use-case) like on the `sidebar` that i've implemented a props. Though i am thinking of putting the references for the functions in the store to make it more accessible but please let me know your thoughts.

Truly sorry for the inconvenience. Didn't mean any of it.

### CODE REVIEW

**Comment by idoshamun** [comment]
Thanks for the quick fix! And don't worry about the inconvenience, it's part of the development.
Two comments though, I would love to see this implementation as part of our Vuex state rather than passing an object between components. Vuex can manage everything and give access to more components to disable keyboard functionality.
Second, I would prefer to use the term keyboard shortcuts/bindings rather than vim navigation.

**Comment by sshanzel** [comment]
Hi Ido! I'll make sure to transfer the commands in the `store` for better accessibility as you mentioned, and i shall rename the term being used later tonight after work.

**Comment by sshanzel** [comment]
Hi Ido, I've already started though i figured that i will be moving quite a few things, and i want it to be very clean, so i might not be able to ship it tonight due to lack of time. Tomorrow is last day of work, which will then i will have more time to continue. Thanks a lot for the patience!

I truly understand the situation as i am also aspiring to release one of my own applications in the near future, please bear with me.

**Comment by idoshamun** [comment]
Good luck with your new application! Let me know if you need any help. No worries, I can also take care of this issue if you find it too much.

**Comment by sshanzel** [comment]
Thanks Ido! Don't worry i'm actually at it right now :P I'm very committed to this package and i'll make sure to give my best! 😁

**Comment by idoshamun** [comment]
Thank you :)

File: packages/extension/src/routes/Home.vue [hunk 1]
```diff
@@ -397,7 +395,7 @@ export default {
   computed: {
     ...mapState('ui', ['notifications', 'showNotifications', 'showSettings', 'theme', 'showDndMenu']),
     ...mapGetters('ui', ['sidebarInstructions', 'showReadyModal', 'dndMode']),
-    ...mapState('feed', ['showBookmarks', 'filter', 'sortBy', 'showFeed']),
+    ...mapState('feed', ['showBookmarks', 'filter', 'sortBy', 'showFeed', 'daFeedRef']),
```
Comment (idoshamun) [inline packages/extension/src/routes/Home.vue]: Why do you need the `daFeedRef` here from the state?
The `Home` component provides it to the state in the first place

**Comment by sshanzel** [comment]
Hi @idoshamun, thanks a lot for the review. The search bar in the feed actually works properly, i think the search bar he's mentioning which i mentioned in the issue, are in the sidebar which needed some work that is included in these commits.

File: packages/extension/src/routes/Home.vue (hunk 1 above)
Comment (sshanzel) [reply packages/extension/src/routes/Home.vue]: Oh, this is not actually being used, sorry for the misplacement!

**Comment by idoshamun** [comment]
Thank you so much for your contribution! Hopefully, I will release it soon :grin:

**Comment by sshanzel** [comment]
Wow, i'm so happy we were able to fix it! Thanks a lot for the patience!

**Comment by idoshamun** [comment]
The fix is available in version 2.14.2

**Pull request merged**